README.md

```diff
@@ -7,51 +7,57 @@ A Swift mixin to use `UITableViewCells`, `UICollectionViewCells` and `UIViewCont
 [![Platform](http://cocoapod-badges.herokuapp.com/p/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
 [![Version](http://cocoapod-badges.herokuapp.com/v/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
 
-## Quick Start
+# Introduction
 
-### Type-safe `UITableViewCell` and `UICollectionViewCell`
-
-* Mark your `UITableViewCell` classes to conform to either `Reusable` or `NibReusable` (no additional code to implement!)
-* Then simply use `tableView.dequeueReusableCell(indexPath: indexPath) as MyCustomCell` to get a dequeued instance of the expected cell class. **No need for you to manipulate `reuseIdentifiers` manually!**
-* Use the same for `UICollectionViewCells`
+This library aims to make it super-easy to create and dequeue/instantiate Reusable views, including `UITableViewCell`, `UICollectionViewCell` and custom `UIView`s, but also VCs from Storyboards… all that by **simply marking your classes as conforming to a protocols, without having to add any code**, and **creating a type-safe API with no more String-based API**.
```

Comment:
Maybe?

This library aims to make it super-easy to create, dequeue and instantiate Reusable views anywhere this pattern is used: From the obvious `UITableViewCell` to `UICollectionViewCell`,  to custom `UIView`s, even supporting `UIViewController`s from Storyboards.